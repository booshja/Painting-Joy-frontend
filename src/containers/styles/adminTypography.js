// dependencies
import styled from "styled-components";

export const StyledP = styled.p`
    font-size: 1.2rem;
    font-family: "News Cycle", sans-serif;
    letter-spacing: 1px;
`;
