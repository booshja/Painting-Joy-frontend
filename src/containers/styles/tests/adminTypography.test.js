import { render } from "@testing-library/react";
import { StyledP } from "../adminTypography";

/** Smoke Test */
it("renders without crashing", () => {
    render(<StyledP>This is a paragraph</StyledP>);
});

/** Snapshot Test */
it("matches snapshot", () => {
    const { asFragment } = render(<StyledP>This is a paragraph</StyledP>);
    expect(asFragment()).toMatchSnapshot();
});

/** RTL Tests */
it("renders correctly", () => {
    const { getByText } = render(<StyledP>This is a paragraph</StyledP>);

    getByText("This is a paragraph");
});
