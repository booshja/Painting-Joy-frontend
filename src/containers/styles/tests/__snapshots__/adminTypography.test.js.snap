// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <p
    class="sc-bdnxRM kQoGtj"
  >
    This is a paragraph
  </p>
</DocumentFragment>
`;
